src/render/portable/mod.rs

```diff
@@ -81,6 +81,11 @@ impl Renderer for ItemRenderer {
                 }
                 // Render table
                 else if let Some(table_specs) = &table.render_table {
+                    let table_specs = &table_specs
+                        .clone()
+                        .into_iter()
+                        .filter(|(_, s)| !s.optional)
```

Comment:
Mhm, not sure I understand this. Optional should behave in the way that the column is used if it is there, but datavzrd does not complain if it is not there. To me, this here looks like everything marked as optional is just ignored here.

Reply:
Phew! Thank god you had a look at this. Seems like my brain had a short dropout 🤦‍♂️ Let me quickly fix that.